Crawlee should not dictate playwright version

### Which package is this bug report for? If unsure which one to select, leave blank

@crawlee/playwright (PlaywrightCrawler)

### Issue description

I need features from 1.29.0 and can't use them because https://github.com/apify/crawlee/blob/6781f121ccb413958d09c1288a1e1e9463783c3c/packages/playwright-crawler/package.json#L67

This is a serious limitation.

Questions:

1.) Will you raise the playwright version to 1.29 in the next weeks?
2.) What is the policy for your playwright version updates?
3.) Why is crawlee so strict about playwright versions in the first place?

Thanks,
```
npm WARN ERESOLVE overriding peer dependency
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: crawlee@3.1.4
npm ERR! Found: playwright@1.29.0
npm ERR! node_modules/playwright
npm ERR!   peerOptional playwright@">= 1.21.x <= 1.27.x" from @crawlee/playwright@3.1.4
npm ERR!   node_modules/crawlee/node_modules/@crawlee/playwright
npm ERR!     @crawlee/playwright@"^3.1.4" from crawlee@3.1.4
npm ERR!     node_modules/crawlee
npm ERR!       crawlee@"3.1.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional playwright@">= 1.21.x <= 1.27.x" from crawlee@3.1.4
npm ERR! node_modules/crawlee
npm ERR!   crawlee@"3.1.4" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: playwright@1.27.1
npm ERR! node_modules/playwright
npm ERR!   peerOptional playwright@">= 1.21.x <= 1.27.x" from crawlee@3.1.4
npm ERR!   node_modules/crawlee
npm ERR!     crawlee@"3.1.4" from the root project

### Code sample

```TypeScript
{
  "name": "crawlee-reproduction",
  "type": "module",
  "dependencies": {
    "crawlee": "3.1.4",
    "playwright": "1.29.0"
  }
}
```


### Package version

3.1.4

### Node.js version

v16.13.1

### Operating system

Ubuntu 20.04.5

### Apify platform

- [ ] Tick me if you encountered this issue on the Apify platform

### I have tested this on the `next` release

_No response_

### Other context

_No response_

--- a/package.json
+++ b/package.json
@@ -92,7 +92,7 @@
         "lerna": "^6.0.3",
         "lint-staged": "^13.0.3",
         "node-gyp": "^9.1.0",
-        "playwright": "1.27.1",
+        "playwright": "1.29.2",
         "portastic": "^1.0.1",
         "proxy": "^1.0.2",
         "puppeteer": "19.2.0",

--- a/packages/browser-pool/package.json
+++ b/packages/browser-pool/package.json
@@ -51,7 +51,7 @@
         "tslib": "^2.4.0"
     },
     "peerDependencies": {
-        "playwright": ">= 1.21.x <= 1.27.x",
+        "playwright": "<= 2.x",
         "puppeteer": "<= 19.x"
     },
     "peerDependenciesMeta": {

--- a/packages/crawlee/package.json
+++ b/packages/crawlee/package.json
@@ -69,7 +69,7 @@
         "import-local": "^3.1.0"
     },
     "peerDependencies": {
-        "playwright": ">= 1.21.x <= 1.27.x",
+        "playwright": "<= 2.x",
         "puppeteer": "<= 19.x"
     },
     "peerDependenciesMeta": {

--- a/packages/playwright-crawler/package.json
+++ b/packages/playwright-crawler/package.json
@@ -64,7 +64,7 @@
         "ow": "^0.28.1"
     },
     "peerDependencies": {
-        "playwright": ">= 1.21.x <= 1.27.x"
+        "playwright": "<= 2.x"
     },
     "peerDependenciesMeta": {
         "playwright": {

--- a/test/e2e/playwright-chromium-experimental-containers/actor/main.js
+++ b/test/e2e/playwright-chromium-experimental-containers/actor/main.js
@@ -1,6 +1,9 @@
 import { Actor } from 'apify';
 import { Dataset, PlaywrightCrawler } from '@crawlee/playwright';
+import { skipTest } from '../../tools.mjs';
 import { ApifyStorageLocal } from '@apify/storage-local';
+
+skipTest('fails after update to playwright 1.29.0, looks like issue the chromium extension, maybe the manifest_version 2 vs 3?');
 
 const mainOptions = {
     exit: Actor.isAtHome(),

